examples/webgl_shadow_contact.html

```diff
@@ -0,0 +1,279 @@
+				shadowGroup.position.y = - 0.3;
+				scene.add( shadowGroup );
+
+				// the render target that will show the shadows in the plane texture
+				renderTarget = new THREE.WebGLRenderTarget( 512, 512 );
+				renderTarget.texture.generateMipmaps = false;
+
+				// the render target that we will use to blur the first render target
+				renderTargetBlur = new THREE.WebGLRenderTarget( 512, 512 );
+				renderTargetBlur.texture.generateMipmaps = false;
+
+
+				// make a plane and make it face up
+				planeGeometry = new THREE.PlaneGeometry( PLANE_WIDTH, PLANE_HEIGHT ).rotateX( Math.PI / 2 );
+				planeMaterial = new THREE.MeshBasicMaterial( {
+					map: renderTarget.texture,
+					opacity: state.opacity,
+					transparent: true,
+				} );
+				plane = new THREE.Mesh( planeGeometry, planeMaterial );
+				shadowGroup.add( plane );
+
+				// the y from the texture is flipped!
+				plane.scale.y = - 1;
+
+				// the plane onto which to blur the texture
+				blurPlane = new THREE.Mesh( planeGeometry );
+				blurPlane.visible = false;
+				shadowGroup.add( blurPlane );
+
+				// the camera to render the depth material from
+				shadowCamera = new THREE.OrthographicCamera( - PLANE_WIDTH / 2, PLANE_WIDTH / 2, PLANE_HEIGHT / 2, - PLANE_HEIGHT / 2, 0, CAMERA_HEIGHT );
+				shadowCamera.rotation.x = Math.PI / 2; // get the camera to look up
+				shadowGroup.add( shadowCamera );
+
+				cameraHelper = new THREE.CameraHelper( shadowCamera );
+
+				// like MeshDepthMaterial, but goes from black to transparent
+				depthMaterial = new THREE.ShaderMaterial( {
+					uniforms: {
+						darkness: { value: state.darkness },
+					},
+					vertexShader: THREE.ShaderChunk[ 'depth_vert' ],
+					fragmentShader: `
+						#define DEPTH_PACKING 3200
+						uniform float darkness;
+						${THREE.ShaderChunk[ 'depth_frag' ].replace(
+					'gl_FragColor = vec4( vec3( 1.0 - fragCoordZ ), opacity );',
+					'gl_FragColor = vec4( vec3( 0.0 ), ( 1.0 - fragCoordZ ) * darkness );'
+				)}
+					`,
+				} );
+				depthMaterial.depthTest = false;
+				depthMaterial.depthWrite = false;
+
+				horizontalBlurMaterial = new THREE.ShaderMaterial( HorizontalBlurShader );
+				horizontalBlurMaterial.depthTest = false;
+
+				verticalBlurMaterial = new THREE.ShaderMaterial( VerticalBlurShader );
+				verticalBlurMaterial.depthTest = false;
+
+				//
+
+				gui = new GUI();
+
+
+				gui.add( state, 'blur', 0, 15, 0.1 );
+				gui.add( state, 'darkness', 1, 5, 0.1 ).onChange( function () {
+
+					depthMaterial.uniforms.darkness.value = state.darkness;
+
+				} );
+				gui.add( state, 'opacity', 0, 1, 0.01 ).onChange( function () {
+
+					planeMaterial.opacity = state.opacity;
+
+				} );
+				gui.add( state, 'showWireframe', true ).onChange( function () {
+
+					if ( state.showWireframe ) {
+
+						scene.add( cameraHelper );
+
+					} else {
+
+						scene.remove( cameraHelper );
+
+					}
+
+				} );
+
+				//
+
+				renderer = new THREE.WebGLRenderer( { antialias: true, alpha: true } );
```

Comment:
I don't think it's necessary to set `alpha` to `true`...

Reply:
Yes, it's to show that the shadows rendered are transparent (i was using them in AR).
The white you see is the white of the body.

Should I use `renderer.setClearColor(0xffffff, 1)` instead?

Reply:
Try it with `scene.background = new THREE.Color( 0xffffff );`